CRM/Utils/System/Base.php

```diff
@@ -705,7 +705,7 @@ public function getTimeZoneOffset() {
 
       $timeZoneOffset = sprintf("%02d:%02d", $tz / 3600, abs(($tz / 60) % 60));
 
-      if ($timeZoneOffset > 0) {
+      if ($tz >= 0) { // $tz is a number, not a string, so it is better to test it.  00:00 needs a plus sign.
```

Comment:
It is not just a tidy up, notice I changed > to >= because when in the Europe/London time zone it was returning 00:00, not +00:00 and this caused problems later.  By all means remove the comments.  As you probably know, the offset to UTC in Europe/London is 00:00 in winter and +01:00 in summer.  As I am in that time zone, when I was investigating, I hit the problem.